scripts/lib/CIME/case/case_submit.py

```diff
@@ -123,8 +123,6 @@ def submit(self, job=None, no_batch=False, prereq=None, resubmit=False,
     if resubmit and os.path.exists(submit_options):
         config = configparser.SafeConfigParser()
         config.read(submit_options)
-        if not skip_pnl and config.has_option('SubmitOptions','skip_pnl'):
```

Comment:
What's the reason for this change?

Reply:
the skip_pnl is a logical and needs to be converted to text to write, I'm not sure why this didn't fail in the original PR but it's failing now.